### ISSUE TITLE
Server crahses if database goes offline

### ISSUE THREAD

**Issue by tomascw** [mysimulation/MySim#11]
Currently the server crashes if the database goes offline or changes IP. Instead the server should check the server's heartbeat, similar to issue #5. A temporary solution is to connect the database server from a domain.

**Comment by jackmawer** [comment]
Would it be possible to have a fallback database, or have a backup database spin up locally to keep the server going?

**Comment by tomascw** [comment]
Yes. We can provide a "ORM" abstraction layer that can save data in both MySQL and SQLite formats. We can fallback on SQLite if MYSQL fails.
